src/routes/welcome/components/Layout.tsx

```diff
@@ -61,30 +113,24 @@ export const LoadSafe = ({ provider, size }) => (
 const Welcome = ({ isOldMultisigMigration, provider }: any) => {
   const headingText = isOldMultisigMigration ? (
     <>
-      We will replicate the owner structure from your existing Gnosis MultiSig
-      <br />
-      to let you test the new interface.
-      <br />
-      As soon as you feel comfortable, start moving funds to your new Safe.
-      <br />{' '}
+      We will replicate the owner structure from your existing Gnosis MultiSig to let you test the new interface. As
+      soon as you feel comfortable, start moving funds to your new Safe.
     </>
   ) : (
     <>
-      Gnosis Safe Multisig is the most secure way to manage crypto funds
-      <br />
-      collectively. It is an improvement of the Gnosis MultiSig, which is used by more than 3000 teams
-      <br /> and stores over $1B USD worth of digital assets. Gnosis Safe Multisig features a modular
-      <br /> design, formally verified smart contracts and vastly improved user experience.{' '}
+      Gnosis Safe Multisig is the most trusted platform to manage <br /> digital assets on Ethereum. Here is how to get
```

Comment:
Can we change this to:


`Gnosis Safe Multisig is the most trusted platform to manage digital assets. <br /> Here is how to get started:{' '}`

As Safe isn't only available on Ethereum anymore.